src/components/Tutorial.jsx

```diff
@@ -13,8 +13,12 @@ options.log = (test) => {
 function Tutorial(props) {
   const { contents } = props;
   const tutorial = contents.original || { steps: [] };
+  const translation = contents.translation || { steps: [] };
   const steps = [];
-  tutorial.steps && tutorial.steps.map((step, key) => steps.push(<p key={key}><b>{key}</b> {step.content}</p>));
+  tutorial.steps && tutorial.steps.map((step, key) => {
+    steps.push(<p data-translation-key={`steps.${key}.content`} key={key}><b>{key}</b> {step.content}</p>);
+    translation.steps[key] && steps.push(<p key={`translation-${key}`}><b>translation</b> {translation.steps[key].content}</p>);
+  });
```

Comment:
No harm in translating it, I suppose, but tutorial titles are only displayed in the project builder.